translations/harbour-seaprint-de.ts

```diff
@@ -613,1234 +613,1201 @@ auf diesem Drucker</translation>
     </message>
     <message>
         <source>plaintext</source>
-        <translation type="unfinished"></translation>
+        <translation>Klartext</translation>
     </message>
     <message>
         <source>SVGs</source>
-        <translation type="unfinished"></translation>
+        <translation>SVGs</translation>
     </message>
 </context>
 <context>
     <name>strings</name>
     <message>
         <source>28 x 40″</source>
-        <translation type="unfinished"></translation>
+        <translation>28 x 40 Zoll</translation>
     </message>
     <message>
         <source>2a0</source>
-        <translation type="unfinished"></translation>
+        <translation>2a0</translation>
     </message>
     <message>
         <source>A0</source>
-        <translation type="unfinished"></translation>
+        <translation>DIN A0</translation>
```

Comment:
Does the DIN prefix really matter? Aren't A-sizes ISO since forever?
If so, add it to all, e.g. 2a0.... but i suggest it is dropped.

Reply:
Ok, Drop it